contracts/SmartLoansFactory.sol

```diff
@@ -53,7 +53,7 @@ contract SmartLoansFactory is IBorrowersRegistry {
   }
 
   function createLoan() external oneLoanPerOwner returns(SmartLoan) {
```

Comment:
Hmm, we should rather use Factory owner as an initial governor. 
Leaving governing right to users who create loans will give them ablitity to remove solvency restrictions.